sky/packages/sky/lib/src/widgets/framework.dart

```diff
@@ -1259,7 +1259,7 @@ abstract class OneChildRenderObjectWrapper extends RenderObjectWrapper {
 
   void syncRenderObject(RenderObjectWrapper old) {
```

Comment:
you should change the signature of the method to take a OneChildRenderObjectWrapper and drop the cast on the line you modified